include/language/en_us.lang.php

```diff
@@ -3034,6 +3034,19 @@
 $app_list_strings['aor_assign_options']['all'] = 'ALL Users';
 $app_list_strings['aor_assign_options']['role'] = 'ALL Users in Role';
 $app_list_strings['aor_assign_options']['security_group'] = 'ALL Users in Security Group';
+$app_list_strings['template_date_format'][''] = '';
```

Comment:
Hi @jack7anderson7,

We should probably initialized this entry with something like `$app_list_strings['template_date_format'] = []`